crates/uv-resolver/src/python_requirement.rs

```diff
@@ -1,32 +1,31 @@
-use pep440_rs::Version;
-use pep508_rs::MarkerEnvironment;
+use pep508_rs::{MarkerEnvironment, StringVersion};
 use uv_interpreter::Interpreter;
 
-#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd)]
+#[derive(Debug, Clone, Eq, PartialEq)]
 pub struct PythonRequirement {
     /// The installed version of Python.
-    installed: Version,
+    installed: StringVersion,
     /// The target version of Python; that is, the version of Python for which we are resolving
     /// dependencies. This is typically the same as the installed version, but may be different
     /// when specifying an alternate Python version for the resolution.
-    target: Version,
+    target: StringVersion,
 }
 
 impl PythonRequirement {
     pub fn new(interpreter: &Interpreter, markers: &MarkerEnvironment) -> Self {
         Self {
-            installed: interpreter.python_version().clone(),
-            target: markers.python_full_version.version.clone(),
+            installed: interpreter.python_full_version().clone(),
```

Comment:
This is the same value, I just renamed the method because it's the `python_full_version` marker.